editions/test/tiddlers/tests/test-filters.js

```diff
@@ -1009,6 +1009,21 @@ Tests the filtering mechanism.
 			expect(wiki.filterTiddlers("[title[HelloThere]] +[format:titlelist[]]").join(" ")).toBe("HelloThere");		
 		});
 	
+		it("should handle the parsedate operator", function() {
```

Comment:
We should also have tests for trying to convert invalid dates.

Reply:
There is already the test case `expect(wiki.filterTiddlers("[[nodate]parsedate[ISO]]").join(" ")).toBe("");` for an invalid date. I'm using the JavaScript engines `Date` constructor to parse the date. Testing the JavaScript engine by providing different invalid dates was out of scope in my opinion. I just want to test the error handling of TW's implementation of the operator and thought that one test with an (obviously invalid) date would be sufficient.

Reply:
Thanks @FlashSystems